packages/scxa-tsne-plot/src/TSnePlotView.js

```diff
@@ -176,19 +206,17 @@ TSnePlotView.propTypes = {
   wrapperClassName: PropTypes.string,
   clusterPlotClassName: PropTypes.string,
   expressionPlotClassName: PropTypes.string,
+  selectedParameter: PropTypes.number.isRequired,
```

Comment:
What do you think if we rename `selectedParameter` to `selectedPlotOption`. In my opinion, it would be more readable and easy to understand.